src/NuGet.Clients/NuGet.Tools/OutputConsoleLogger.cs

```diff
@@ -51,9 +56,7 @@ public class OutputConsoleLogger : INuGetUILogger, IDisposable
 
             NuGetUIThreadHelper.JoinableTaskFactory.Run(async () =>
```

Comment:
this is problematic to try to switch to UI thread in MEF constructor. Please make `_dte` as `AsyncLazy` or add a separate `initializeAsync` api